mpp-test/src/commonMain/kotlin/arrow/core/Option.kt

```diff
@@ -0,0 +1,158 @@
+package arrow.core
+
+import mkapt.annotations.higherkind
+
+/**
+ * Represents optional values. Instances of `Option`
+ * are either an instance of $some or the object $none.
+ */
+@higherkind
```

Comment:
@higherkind should not be needed any longer right? It generates warnings as it is not supported in all platforms

```
> Task :mpp-test:compileKotlinJs
w: meta/mpp-test/src/commonMain/kotlin/arrow/core/Option.kt: (9, 1): Reflection is not supported in JavaScript target, therefore you won't be able to read this annotation in run-time
```

Is this something we should worry to strip as part of the process? Or will the generated code be moved and published as an independent artifact than arrow core library?

Reply:
on the MPP core, we don't need any meta-annotation because we are copying the generated code so the copy task can delete them